File: data/python/COUNT_ENTRIES_EQUAL_TO_X_IN_A_SPECIAL_MATRIX_MAIN.py
if __name__ == '__main__':
    param = [
    (47,30,),
    (57,16,),
    (55,63,),
    (11,23,),
    (55,49,),
    (63,64,),
    (64,98,),
    (28,30,),
    (49,61,),
    (48,64,)
        ]
    n_success = 0
    for i, parameters_set in enumerate(param):
        try:
            if f_filled(*parameters_set) == f_gold(*parameters_set):
                print("COUNT_ENTRIES_EQUAL_TO_X_IN_A_SPECIAL_MATRIX," + sys.argv[1] + "," + str(i) + ",SUCCESS")
            else:
                print("COUNT_ENTRIES_EQUAL_TO_X_IN_A_SPECIAL_MATRIX," + sys.argv[1] + "," + str(i) + ",FAILURE")
        except:
            print("COUNT_ENTRIES_EQUAL_TO_X_IN_A_SPECIAL_MATRIX," + sys.argv[1] + "," + str(i) + ",EXCEPTION")

File: data/java/COUNT_ENTRIES_EQUAL_TO_X_IN_A_SPECIAL_MATRIX_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<Integer> param0 = new ArrayList<>();
    param0.add(47);
    param0.add(57);
    param0.add(55);
    param0.add(11);
    param0.add(55);
    param0.add(63);
    param0.add(64);
    param0.add(28);
    param0.add(49);
    param0.add(48);
    List<Integer> param1 = new ArrayList<>();
    param1.add(30);
    param1.add(16);
    param1.add(63);
    param1.add(23);
    param1.add(49);
    param1.add(64);
    param1.add(98);
    param1.add(30);
    param1.add(61);
    param1.add(64);
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result\n");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
            builder.append("COUNT_ENTRIES_EQUAL_TO_X_IN_A_SPECIAL_MATRIX," + args[1] + "," + i + "," + f_gold(param0.get(i),param1.get(i)) + "\n");
        } catch (Exception e) {
            builder.append("COUNT_ENTRIES_EQUAL_TO_X_IN_A_SPECIAL_MATRIX," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}
